tests/lib/newlib/heap_listener/src/main.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sys/libc_heap_listener.h>
+#include <zephyr.h>
+#include <ztest.h>
+
+#include <malloc.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stddef.h>
+
+/* Function used by malloc() to obtain or free memory to the system */
+extern void *_sbrk(intptr_t count);
+
+static uintptr_t current_heap_end(void)
+{
+	return (uintptr_t) _sbrk(0);
```

Comment:
Nit: spurious space
```suggestion
	return (uintptr_t)_sbrk(0);
```

Reply:
Ok, done. I thought uncrustify would fix such things, but apparently not.